test/utils.py

```diff
@@ -4,6 +4,7 @@
 import subprocess
 import time
 
+import psutil
```

Comment:
It is not shipped together with Python, it is a separate module. So it should be added to `test/requirements.txt`.

Reply:
[Done](https://github.com/tarantool/tt/blob/igrishnov/gh-71-free-port-autoresolve-for-tests/test/requirements.txt#L7)

Reply:
I overlooked it somehow, sorry.